app/views/layouts/application.html.erb

```diff
@@ -1,18 +1,25 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>JakSieMaszSlowfood</title>
     <% if Rails.env.test? %>
       <%= tag.meta id: :fake_position, content: fake_location %>
     <% end %>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>JakSieMaszSlowfood</title>
+
+    <link href="https://fonts.googleapis.com/css?family=Play" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Raleway|Roboto" rel="stylesheet">
+
     <%= csrf_meta_tags %>
     <%= javascript_include_tag "https://maps.googleapis.com/maps/api/js?key=#{ENV['GEOCODER_APIKEY']}", 'data-turbolinks-track' => 'reload'%>
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_include_tag "https://maps.googleapis.com/maps/api/js?key=#{ENV['GEOCODER_APIKEY']}", 'data-turbolinks-track' => 'reload'%>
```

Comment:
Why are we loading this again when it was done already at line 15? 🤔

Reply:
Because I'm blind and stupid.